src/components/gridContainer.js

```diff
@@ -456,7 +462,66 @@ class GridContainer extends Component {
             this.setState({statusMessage: message})
         }
     }
+
+    // validation methods to determine whether the table meets accessibility standards
+    tableHasAtLeastOneHeaderRowOrColumn() {
+        if (this.state.metaHeadercols == 0 && this.state.metaHeaderrows == 0) {
+            this.onError(no_headers_error_message);
```

Comment:
I was expecting this method to return just a status and the error set centrally but this also does it maybe this can be reworked to a `validate` method which could set the error and this would just return the validation status of this aspect?